### DISCUSSION TITLE
Assets are not injected into error pages

### DISCUSSION THREAD

**Discussion by iammarjamal** [livewire/livewire#6554] [7. Bugs (archived)]
### Livewire version

3.0

### Which PHP version are you using?

PHP 8.2

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Zephni** [comment]
I am experiencing this same issue. Also including:
```php
Livewire::forceAssetInjection();
```
...within AppServiceProvider.php's boot() method isn't working on any exception pages.

### A hacky fix that's horrible but works
1. Make each error page extend a custom error layout blade
```blade
@extends('errors::layout')
```
2. Add the oldschool livewire style and scripts includes on the error layout blade view
```blade
@livewireStyles
@livewireScripts
```

This fixes the issue but feels hacky, no idea why this isn't done automatically on just the exception pages.
